lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0",
```

Comment:
`2.11.0` according to package? (latest version)

Reply:
Good spot!